### PR TITLE
[WIP] Bug fixing before 0.6.6

### PR DESCRIPTION
**Pull request by csoler**

### CODE REVIEW

File: retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp [hunk 1]
```diff
@@ -445,15 +445,15 @@ QVariant RsGxsForumModel::textColorRole(const ForumModelPostEntry& fmpe,int /*co
     if( (fmpe.mPostFlags & ForumModelPostEntry::FLAG_POST_IS_MISSING))
         return QVariant(mTextColorMissing);
 
+    if(fmpe.mPostFlags & ForumModelPostEntry::FLAG_POST_IS_PINNED)
```
Comment (chelovechishko) [inline retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp:448]: Why pinned post couldn't be in (un)read shape (via color change) at all?

File: retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp [hunk 2]
```diff
@@ -595,8 +595,8 @@ QVariant RsGxsForumModel::pinnedRole(const ForumModelPostEntry& fmpe,int /*colum
 
 QVariant RsGxsForumModel::backgroundRole(const ForumModelPostEntry& fmpe,int /*column*/) const
 {
-    if(fmpe.mPostFlags & ForumModelPostEntry::FLAG_POST_IS_PINNED)
-        return QVariant(QBrush(mBackgroundColorPinned));
+//    if(fmpe.mPostFlags & ForumModelPostEntry::FLAG_POST_IS_PINNED)
+//        return QVariant(QBrush(mBackgroundColorPinned));
```
Comment (chelovechishko) [inline retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp:599]: Then in stylesheets qproperty need to be commented out too?

File: retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp (hunk 1 above)
Comment (csoler) [reply retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp:448]: It doesn't mean much that a pinned post is unread or not, and the place to put the unread icon is occupied by the pin icon. Anyway, it's still possible to use bold of course.

Comment (chelovechishko) [reply retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp:448]: From my point of view whether post (any) read or not is quite important. When there are many pinned posts and one of them not yet read, guessing what exactly is, is not something good for me (whether i just have not time to read yet, or i saved it in unread state for the purpose).

Comment (csoler) [reply retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp:448]: Agreed. Thx for checking.

Comment (chelovechishko) [reply retroshare-gui/src/gui/gxsforums/GxsForumModel.cpp:448]: Thx for agreeding.

**Comment by defnax** [comment]
Im wondering why you removed the background color feature for pinned posts? it was nice feature where the color can changed from the qss files standard and darkstyles. which was added by @chelovechishko 

but only the hardcoded colors at the beginning where you added that was not so good.

**Comment by csoler** [comment]
Seemed like a legitimate request to me. In the default style at least, it looks pretty bad. Having it using an icon seemed like a better idea.

**Comment by defnax** [comment]
on my changes i added here too before it doesnt worked not sure if its needed 
image

now i must wait till your PR is merged? my comments changes depends on this PR.

**Comment by csoler** [comment]
@defnax: I will merge this asap. Just waiting for appveyor checks to complete.

**Pull request merged**